dojoWordle/components/Game/Game.tsx

```diff
@@ -1,18 +1,42 @@
-import React from "react";
+import React, { useState } from "react";
+
 import { View } from "react-native";
 import { Keyboard } from "../Keyboard/Keyboard";
-import { LetterBox } from "../LetterBox.tsx/LetterBox";
-import { ResultContainer } from "./Game.style";
+import {
+  ResultContainer,
+  displayCheckedWord,
+  displayTypingWord,
+  wordToFind,
+} from "./Game.style";
 
 export function Game() {
+  const [wordInput, setWordInput] = useState("");
+  const [wordValid, setWordValid] = useState("");
+
+  const keyPress = (key: string) => {
```

Comment:
```suggestion
  const onKeyPress = (key: string) => {
```

By convention, we name event handlers `on${EventName}`